modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -1404,6 +1405,87 @@ public String getName() {
         return visible;
     }
 
+    /**
+     * Whether or not this {@code Node} is showing (that is, it is part of an
+     * open Window on the user's system). The Window might be "showing", yet the user might not
+     * be able to see it due to the Window being rendered behind another Window
+     * or due to the Window being positioned off the monitor.
+     *
+     * <p>Note that the {@code Node} does not need to be visible for this property
+     * to be {@code true}.
```

Comment:
We might also want to add a paragraph explaining its use. Something like:
```
This property can be used in conjunction with {@link ObservableValue#when} as a source for listeners that should be collected when the node is not shown.
```
If there are other uses we can mention them as well.

Reply:
I've changed the documentation as you suggested, and I also included that the `Node` must be part of a `Scene`, although I suppose that is a given since you can't show a node in a window otherwise.

> Also we need to be careful not to confuse shown with showing, which are properties on ComboBox, ChoiceBox and others.
Definitely, I had `showing` in the initial version only to discover it was already present in common controls.

I also added your suggested paragraph but rephrased as:

> This property can be used in conjunction with {@link ObservableValue#when} to create bindings which are only actively listening to their source when the node is shown.

Other uses could be to disable animations when not shown.  For example, `ProgressBarSkin` and some other controls only animate their controls when visible (they however do a visibility check as well).  See `isTreeShowing`.  

For example, I've used this property to start animations as soon as a `Node` becomes shown, and to stop the animation when it is no longer shown.  I've added this to the docs:

> This property can also be used to start animations when the node is shown, and to stop them when it is no longer shown.

I also added a `@see ObservableValue#when(ObservableValue)` -- not sure if that is needed.